src/components/Main/Main.js

```diff
@@ -1,12 +1,56 @@
 import "./Main.css";
 import Form from "../Form/Form";
 import Entries from "../Entries/Entries";
+import useLocalStorageState from "use-local-storage-state";
 
 export default function Main() {
+  const [entries, setEntries] = useLocalStorageState("entries", {
+    defaultValue: [
+      {
+        id: 1000,
+        date: "Feb 5, 2025",
+        motto: "We are in a state of chaos",
+        notes:
+          "Today I learned about React State. It was fun! I can't wait to learn more.",
+      },
+      {
+        id: 999,
+        date: "Feb 4, 2025",
+        motto: "Props, Props, Props",
+        notes:
+          "Today I learned about React Props. Mad props to everyone who understands this!",
+      },
+      {
+        id: 998,
+        date: "Feb 3, 2025",
+        motto: "How to nest components online fast",
+        notes:
+          "Today I learned about React Components and how to nest them like a pro. Application design is so much fun!",
+      },
+      {
+        id: 997,
+        date: "Feb 2, 2025",
+        motto: "I'm a React Developer",
+        notes: "My React-ion when I learned about React: 😍",
+      },
+    ],
+  });
+
+  function handleCreateEntries(motto, notes) {
+    const newEntry = {
+      id: 1001,
```

Comment:
You could pass the "motto" instead the id, so that the key is generated differently with each entry. It's only a workaround but right now, react throws an error beginning with the second user entry: 
<img width="798" alt="Bildschirmfoto 2022-11-23 um 08 09 25" src="https://user-images.githubusercontent.com/115539625/203489634-00cf6311-bd9b-4be2-8b26-2ebca4c9f479.png">

Reply:
Du bist so ein 🦊